src/ik_base.h

```diff
@@ -115,8 +115,8 @@ struct Random
         return random_gauss_buffer + i;
     }
 
-    Random()
-        : rng(std::random_device()())
+    Random(std::minstd_rand::result_type seed = std::random_device()())
+        : rng(seed)
```

Comment:
I would propose to leave the initialization here out altogether.
The only uses of this internal class are through the plugin API which will initialize it randomly (or from parameter).

Reply:
correct but Random is a public API that's why I added it. Anyway, will remove.

Reply:
> but Random is a public API

No, why do you think so? `src/ik_base.h` is not installed, nor included from a header in `include/`.